R/get-osmdata-df.R

```diff
@@ -3,9 +3,9 @@
 #'
 #' @inheritParams osmdata_sp
 #' @param q An object of class `overpass_query` constructed with
-#'      \link{opq} and \link{add_osm_feature}. May be be omitted,
-#'      in which case the attributes of the \link{data.frame} will not include
-#'      the query.
+#'      \link{opq} and \link{add_osm_feature} or a string with a valid query.
```

Comment:
Thanks @jmaspons, this is definitely important. I'd also suggest it should be expanded to:
> a string with a valid query, such as ... \<simple example\>.

Then add another sentence at the end of description for `q`:
> See examples below.

... And then extend the @examples with good examples of all kinds of usage.

Reply:
Ok. I added examples of complex queries which are not possible to build with regular functions, @mpadge